src/runtime/vulkan/vulkan.cc

```diff
@@ -487,15 +488,29 @@ void VulkanDeviceAPI::GetAttr(Device dev, DeviceAttrKind kind, TVMRetValue* rv)
       break;
     }
     case kMaxRegistersPerBlock:
-      return;
+      break;
     case kGcnArch:
-      return;
+      break;
     case kApiVersion:
-      return;
+      *rv = VK_HEADER_VERSION;
+      break;
+    case kDriverVersion: {
+      int64_t value = phy_prop.driverVersion;
+      std::ostringstream os;
+      os << VK_VERSION_MAJOR(value) << "." << VK_VERSION_MINOR(value) << "."
+         << VK_VERSION_PATCH(value);
+      *rv = os.str();
+      break;
+    }
   }
 }
 
 VulkanDeviceAPI::VulkanDeviceAPI() {
+  uint32_t version;
+  vkEnumerateInstanceVersion(&version);
+  std::cout << "Vulkan instance version " << VK_VERSION_MAJOR(version) << "."
```

Comment:
LOG(INFO)

Reply:
Thank you for catching that.  Removed, and PR is updated.